src/rgw/rgw_data_sync.cc

```diff
@@ -2728,43 +2871,46 @@ int RGWRunBucketSyncCoroutine::operate()
     }
     while (!lease_cr->is_locked()) {
       if (lease_cr->is_done()) {
-        ldout(cct, 5) << "lease cr failed, done early" << dendl;
+        tn->log(5, "failed to take lease");
         set_status("lease lock failed, early abort");
         return set_cr_error(lease_cr->get_ret_status());
       }
       set_sleeping(true);
       yield;
     }
 
+    tn->log(10, "took lease");
     yield call(new RGWReadBucketSyncStatusCoroutine(sync_env, bs, &sync_status));
     if (retcode < 0 && retcode != -ENOENT) {
-      ldout(sync_env->cct, 0) << "ERROR: failed to read sync status for bucket="
-          << bucket_shard_str{bs} << dendl;
+      tn->log(0, SSTR("ERROR: failed to read sync status for bucket="
+          << bucket_shard_str{bs}));
       lease_cr->go_down();
       drain_all();
       return set_cr_error(retcode);
     }
 
-    ldout(sync_env->cct, 20) << __func__ << "(): sync status for bucket "
-        << bucket_shard_str{bs} << ": " << sync_status.state << dendl;
+    tn->log(20, SSTR("sync status for bucket "
+        << bucket_shard_str{bs} << ": " << sync_status.state));
```

Comment:
i think we should leave `bucket_shard_str` part out of these messages, now that it's already included in the log message. for example, this one looks like:
> data:sync:shard[75]:entry[eslbge-29:d2560962-42d3-4295-ac0c-efbbce6b79d8.4134.9]:bucket[eslbge-29:d2560962-42d3-4295-ac0c-efbbce6b79d8.4134.9]: sync status for bucket eslbge-29:d2560962-42d3-4295-ac0c-efbbce6b79d8.4134.9: 0